### PR TITLE
Clarify the purpose of ingress network in the documentaiton

### PR DESCRIPTION
**Pull request by sanimej**
Signed-off-by: Santhosh Manohar santhosh@docker.com

### CODE REVIEW

File: docs/swarm/ingress.md [hunk 1]
```diff
@@ -28,6 +29,10 @@ ports open between the swarm nodes before you enable swarm mode:
 You must also open the published port between the swarm nodes and any external
 resources, such as an external load balancer, that require access to the port.
 
+**Note** `ingress` network exists only to route the external traffic to the right
```
Comment (thaJeztah) [inline docs/swarm/ingress.md]: _The_ ingress network?

Also this should be formatted as a note;

```Markdown
> **Note**: The `ingress` network....
```

**Review by thaJeztah — CHANGES_REQUESTED** [review]
One nit, but otherwise looks good; however this should now be opened in the https://github.com/docker/docker.github.io repository. I'm ok with merging it here as well for as long as those files are still here

**Comment by vdemeester** [comment]
ping @sanimej

**Comment by sanimej** [comment]
@thaJeztah Looks like the doc files have been moved over to docker.github.io. I will close this and open in the doc repo.

**Comment by thaJeztah** [comment]
Thanks @sanimej

**Pull request closed without merging**